modules/controller/index.js

```diff
@@ -42,84 +53,27 @@ Controller.prototype._optsOrDefault = function(opts, method) {
   });
 };
 
-Controller.prototype._validateInit = function(opts) {
-  var source = this.source;
-
-  filterInitValidator(opts, source);
-  relationsInitValidator(opts, source);
-  modelMethodInitValidator(opts, source);
-};
-
 Controller.prototype._buildRequestHandler = function(opts) {
-  var source = this.source;
   var responder = opts.responder || this.responder;
   var method = opts.controllerMethod;
 
-  var validateRequest = this._lookupValidateRequest(method);
-  var polyResponse = this._lookupPolyResponse(method);
-  var handle = this._lookupHandler(method).bind(this);
+  var payload = payloads[method];
+  var handle = this['_' + method + 'Handler'].bind(this);
 
   return function(request, response, next) {
-    var err = validateRequest(request, response, source);
+    var err = contentType(request, response);
```

Comment:
This check only needs to happen on create/update: whenever the client sends a payload to the server. Not required on read or deletes without bodies.